### PR TITLE
#trivial Add storybook to the master branch

### PR DESCRIPTION
**Pull request by thayannevls**
#### What is the purpose of this pull request?
This is a [previously work done](https://github.com/vtex/styleguide/pull/1158) to the branch `features`, but since this branch has been discontinued I'm moving to the `master`.

#### What problem is this solving?
We need an environment with hot-loading and isolated components builds to help us develop our components.

#### How should this be manually tested?
`yarn && yarn storybook`

Visit http://localhost:6006/

#### Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Comment by thayannevls** [comment]
@matheusps Already migrated to v6.x :smile:

**Comment by matheusps** [comment]
Oh, I see. So, why knoobs and not controls ?

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -3,14 +3,46 @@
...
+•
+└── react
+    └── components
+        └── Button
+            ├── index.tsx
+            ├── index.test.tsx
+            └── button.stories.tsx
+```
+
+#### Conventions
```
Comment (matheusps) [inline .github/CONTRIBUTING.md:39]: ✨

**Comment by thayannevls** [comment]
@matheusps Thanks for the reminder! I forgot about the knobs and actions :smile:  could you check again?

**Review by matheusps — APPROVED** [review]
Looks awesome

**Pull request merged**